spyder/widgets/mixins.py

```diff
@@ -1359,9 +1363,17 @@ def set_help_enabled(self, state):
         self.help_enabled = state
 
     def inspect_current_object(self):
-        text = self.get_current_word(help_req=True)

```

Comment:
Is `req=True` still necessary in `get_current_word`? If not, please remove it.

Reply:
Yep the `help_req` is needed for the help request made from the `Editor`